web/crux-ui/src/hooks/use-filters.ts

```diff
@@ -67,3 +67,24 @@ export const textFilterFor =
       return properties.filter(it => !!it && it.toLowerCase().includes(text)).length > 0
     })
   }
+
+export type DateRangeFilter = {
+  dateRange: [Date, Date]
+}
+
+export const dateRangeFilterFor =
+  <Item>(propertiesOf: PropertyValuesOf<Item>): FilterFunction<Item, DateRangeFilter> =>
+  (items: Item[], filter: DateRangeFilter) => {
+    if (filter.dateRange[0] === null && filter.dateRange[1] === null) {
+      return items
+    }
+
+    const from = filter.dateRange[0] === null ? new Date(-8640000000000000) : filter.dateRange[0] // Min Date
```

Comment:
Could you eliminate these numbers somehow? Looks like magic numbers, or outsource to const please.